File: algorithms/1.TwoSum/TwoSum.cpp
// LeeCode 
// 1. TwoSum

class Solution {
public:
    vector<int> twoSum(vector<int>& nums, int target) {
        int sum = 0; 
        for(int i=0; i<nums.size();i++){
           sum = nums[i] + nums[i+1];
            if(sum == target){
               return {i,i+1};
            }
        }// end for 
        return {};
    }
};

File: algorithms/1.TwoSum/TwoSum.swift
class Solution {
    func twoSum(_ nums: [Int], _ target: Int) -> [Int] {
        var sum: Int = 0
        
        for i in 0..<nums.count{
            sum = nums[i] + nums[i+1]
            if(sum == target){
                return [i,i+1]
            }
        }
        return []
    }
}
